genie-web/src/main/java/com/netflix/genie/web/configs/PropertiesConfig.java

```diff
@@ -77,4 +79,18 @@ public HealthProperties healthProperties() {
     public S3FileTransferProperties s3FileTransferProperties() {
         return new S3FileTransferProperties();
     }
+
+    /**
+     * Subset of properties concerned with per-user number of active jobs.
+     * Notice the 'prototype' scope. If this bean is injected via {@code javax.inject.Provider} or Spring's
+     * {@code ObjectFactory}, then values are bound every time, i.e., fast/dynamic properties behavior.
+     *
+     * @return The active job user limit property structure
+     */
+    @Bean
+    @Scope("prototype")
+    @ConfigurationProperties("genie.jobs.users.activeLimit")
```

Comment:
To be clear here you made it possible to completely turn off the feature dynamically instead of just making the `override` map dynamic? I ask cause above it means you have to call `get()` every time just to check enabled. Probably not a huge deal just wanted to be clear

Reply:
Correct.